google-cloud-spanner/src/main/resources/com/google/cloud/spanner/connection/ClientSideStatements.json

```diff
@@ -5,6 +5,7 @@
 	  "name": "SHOW VARIABLE AUTOCOMMIT",
 	  "executorName": "ClientSideStatementNoParamExecutor",
 	  "resultType": "RESULT_SET",
+		"statementType": "SHOW_AUTOCOMMIT",
```

Comment:
nit: we seem to have added extra indentation to all modified lines in this file, please check.

Reply:
Hmmm... That file needs a small cleanup. It uses both tabs AND spaces for indentation (first indentation is one tab, then second indentation is two spaces).
I've made this change consistent with the current format of the file. Let's do the cleanup in a separate PR.